apps/web/__e2e__/keyboard-list-navigation.test.ts

```diff
@@ -255,3 +256,66 @@ test("select notes using Shift+Click upwards", async ({ page }, info) => {
   }
   expect(await notesList[0].isFocused()).toBeTruthy();
 });
+
+test("Ctrl+Click to select unselect notes", async ({ page }, info) => {
+  info.setTimeout(60 * 1000);
+  const { notesList, notes } = await populateList(page, 10);
+  await notes.focus();
+
+  await page.keyboard.down("Control");
+  for (let i = 2; i < 10; i += 2) {
+    await notesList[i].click();
+  }
+  await page.keyboard.up("Control");
+
+  for (let i = 2; i < 10; i += 2) {
+    expect(await notesList[i].isSelected()).toBeTruthy();
+  }
+
+  await page.keyboard.down("Control");
+  for (let i = 2; i < 10; i += 2) {
+    await notesList[i].click();
+  }
+  await page.keyboard.up("Control");
+
+  for (let i = 2; i < 10; i += 2) {
+    expect(await notesList[i].isSelected()).toBeFalsy();
+  }
+});
+
+test.only("Right Click notes", async ({ page }, info) => {
+  info.setTimeout(60 * 1000);
+  const { notesList, notes } = await populateList(page, 10);
+  await notes.focus();
+
+  const noteSelected = 4;
+
+  await notesList[0].click();
+  await notesList[noteSelected].contextMenu.open();
+  expect(
+    (await notesList[noteSelected].contextMenu.title()) ===
```

Comment:
Instead of using `notesList[noteSelected]` again and again, how about assigning it to a variable `selectedNote` or something?